### ISSUE TITLE
Allow Customization of Learn More Text in Info Box Block

### ISSUE THREAD

**Issue by cessnar516** [kadencewp/kadence-blocks#219]
The text for the "Learn More" links in the info box block should be customizable to make the links accessible for users with disabilities. This issue is related to [WCAG Success Criterion 2.4.4: Link Purpose (in Context)](https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-in-context.html). If you have more than one info box on a page and the text for all the links is "Learn More", screen reader users will not be able to distinguish between the links.
